src/DiffOp/Diff/Diff.php

```diff
@@ -395,7 +397,7 @@ public function append( $value ) {
 	 * @param int|string $index
 	 * @param mixed $value
 	 */
-	public function offsetSet( $index, $value ) {
+	public function offsetSet( $index, $value ): void {
```

Comment:
Is this related?

Reply:
The [same](https://github.com/wmde/Diff/pull/129#discussion_r843827883) like for `append()`

Reply:
`#[\ReturnTypeWillChange]` here too?